content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,87 @@
+- **Added** the `ca.istio.io/env` and `ca.istio.io/override` labels for operators to control service account secret generation by namespace.
+- **Added** SDS support to deliver key and certificates to the Istio control plane.
+- **Improved** the ingress gateway to [load the trusted CA certificate from a separate secret](/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-tls-ingress-gateway) when using SDS.
+- **Improved** SDS security by [enforcing the usage of the Kubernetes Trustworthy JWTs](/blog/2019/trustworthy-jwt-sds).
+- **Removed** support for Istio SDS when using [Kubernetes versions earlier than 1.13](/blog/2019/trustworthy-jwt-sds).
+- **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration a future release.
+- **Enabled** the Envoy JWT filter by default to improve security and reliability.
+
+## Telemetry
+
+- **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
+- **Added** alpha support for environmental metadata in Envoy node metadata.
+- **Added** alpha support for Proxy Metadata Exchange.
+- **Added** alpha support for the OpenCensus trace driver.
+- **Improved** reporting for external services by removing requirements to add a service entry.
+- **Improved** the mesh dashboard to provide monitoring of Istio's configuration state.
+- **Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+- **Removed** deprecated `Adapter` and `Template` custom resource definitions (CRDs).
+- **Deprecated** Mixer adapters. Please begin the transition of all extensions to the new extension model. Legacy Mixer integration support continues only for Istio 1.3 and 1.4.
+- **Deprecated** the HTTP API spec used to produce API attributes. We will remove support for producing API attributes in Istio 1.4.
+
+## Policy
+
+- **Improved** rate limit enforcement to allow communication when the quota backend is unavailable.
+
+## Configuration Management
+
+- **Added** a configuration for Galley to use new configuration processing.
+- **Fixed** Galley to stop too many gRPC pings from closing connections.
+- **Improved** Galley to avoid control plane upgrade failures.
+
+## istioctl
+
+- **Added** `istioctl experimental describe pod` to describe an Istio pod's configuration.
```

Comment:
For all these istioctl commands, provide links to the command's reference page.

Reply:
will do in a followon commit after this pass at the review comments.